miner/taiko_worker.go

```diff
@@ -183,6 +183,10 @@ func (w *worker) sealBlockWith(
 				return nil, err
 			}
 		}
+		// Ignore blob txs.
+		if tx.Type() == types.BlobTxType {
+			continue
```

Comment:
Adding a debug log here would be better.

Reply:
Agree.

Reply:
Added debug log.